internal/integration/reql_tests/reql_datum_string_test.go

```diff
@@ -56,7 +56,7 @@ func (suite *DatumStringSuite) TearDownSuite() {
 }
 
 func (suite *DatumStringSuite) TestCases() {
-	suite.T().Log("Running DatumStringSuite: Tests of converstion to and from the RQL string type")
+	suite.T().Log("Running DatumStringSuite: Tests of conversation to and from the RQL string type")
```

Comment:
This should be conversion, not conversation.

```suggestion
	suite.T().Log("Running DatumStringSuite: Tests of conversion to and from the RQL string type")
```

Reply:
done